src/components/Feed.vue

```diff
@@ -35,14 +35,15 @@ export default {
 @import 'components/buttons/btn-5';
 
 .feed-content {
-  width: 100%;
-  margin: 60px auto;
+  margin: 0 auto;
   max-width: 1080px;
+  padding: 80px 20px 20px 80px;
   width: 100%;
 }
 
 .components-feed {
   display: flex;
```

Comment:
If you use a generic class `row` then all of this is set automatically. Also, wrap is the default value.